### ISSUE TITLE
Change over to be built on Ember 2 and pods

### ISSUE THREAD

**Issue by GA-MEB** [ga-wdi-boston/ember-inspector#1]
Images need to be updated as well.

See https://github.com/ga-wdi-boston/ember-object-computed-properties/issues/1

**Comment by GA-MEB** [comment]
Handled in c118803

**Comment by GA-MEB** [comment]
Not resolved - using Ember 2, but not set up with pods.

**Comment by RealWeeks** [comment]
@MattB-GA-Boston is this necessary since they don't look at the code directly and it's a one route app?

**Comment by gaand** [comment]
Is the route generated?

**Comment by RealWeeks** [comment]
@gaand I would think @GA-MEB generated the route but we don't touch code for this training.  Does generating the route change the way `ember inspector` displays information?

**Comment by RealWeeks** [comment]
@GA-MEB does 013/master satisfy this? Still no pod structure, but since they don't see the code didn't update.

**Comment by payne-chris-r** [comment]
I have an idea. What if we use this as a teachable moment and DO show them the code in a place that it isn't actually important. "Hey! Guess what? THIS isn't pod structure. YES you might see this out in the real world, YES you now have a repo to play around with if you want to try and figure out how the other half lives. NO we're not going to teach it you. This is only for a reference." I think there are pedagogical reasons to keep it if we frame it in the right light. @ga-wdi-boston/core ?

**Comment by MicFin** [comment]
@payne-chris-r has there been any decision made on this?

**Comment by payne-chris-r** [comment]
I don't think so. I think similar to the a comment I just wrote [on `Ember`](https://github.com/ga-wdi-boston/ember/issues/16), now that we're doing `default`, it's less of a problem

**Comment by gaand** [comment]
@ga-wdi-boston/core I suggest that this is now obsolete and should be closed.
